### PR TITLE
feat: change AVS list content based on selected AVS

### PR DESCRIPTION
**Pull request by kushagrasarathe**

### CODE REVIEW

**Review by rubo — COMMENTED** [review]
Please add a hover color (either `default` or `content1`) change to make it more interactive.

**Comment by kushagrasarathe** [comment]
> default
@rubo  should also add a active color to show current selected AVS?

**Comment by rubo** [comment]
Yes, please.

File: src/avs/AVSList.jsx [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
-import { useEffect } from 'react';
+import { useEffect, useState } from 'react';
 import { useMutativeReducer } from 'use-mutative';
 import { useServices } from '../@services/ServiceContext';
 import { formatEther } from 'ethers';
 
 export default function AVSList({ onSelectionChange }) {
   const { avsService } = useServices();
   const [state, dispatch] = useMutativeReducer(reduceState, {});
+  const [selectedAVS, setSelectedAVS] = useState(null);
```
Comment (rubo) [inline src/avs/AVSList.jsx:10]: For the state updates, see the line above. We use mutative. As this view grows, we don't wanna have too many `useState.` If you need help with it, just let me know.

**Review by rubo — CHANGES_REQUESTED** [review]

**Review by rubo — APPROVED** [review]

**Pull request merged**